source/whats-new.txt

```diff
@@ -40,12 +40,27 @@ What's New in 4.1
 
 The v4.1 driver release includes the following new features:
 
+- Replaces the ``storage_options()`` method with ``storage_engine()`` in ``mongocxx::v_noabi::options::index``
```

Comment:
```suggestion
- Deprecates ``storage_options()`` method for the newly added ``storage_engine()`` method in ``mongocxx::v_noabi::options::index``
```

Suggest avoiding "Replaces" since `storage_options()` is not-yet removed (only deprecated).